starfish/multiprocessing/pool.py

```diff
@@ -0,0 +1,20 @@
+import multiprocessing.pool as mp
+from typing import Optional
+
+
+class Pool(mp.Pool):
+    """Wrapper class for multiprocessing pool. If n_processes=1 just use map on main
+    thread for debugging purposes """
+
+    def __init__(self, processes: Optional[int]=None, **kwargs):
+        mp.Pool.__init__(self, **kwargs)
+        self.n_processes = processes
+
+    def map(self, func, iterable, chunksize=None):
+        if self.n_processes == 1:
+            self._initializer(*self._initargs)
+            return map(func, iterable)
+        return mp.Pool.map(self, func, iterable, chunksize)
+
+    def __reduce__(self):
```

Comment:
Why is this needed?

Reply:
it's one of the abstract methods required for inheriting Pool

Reply:
I'm confused.  It looks to be [implemented to prevent pool objects from being pickled](https://github.com/python/cpython/blob/master/Lib/multiprocessing/pool.py#L560).  When are we pickling?